### ISSUE TITLE
Usage in other systems - SWADE

### ISSUE THREAD

**Issue by Raileth** [SvenWerlen/fvtt-data-toolbox#10]
Hi,

Feel free to close this if its not a real issue - but I cannot get the module to work in the SWADE ruleset. I have just tried it for items but so far get an error each time I have tried. As per the instructions I have extracted an item I created, and used that for the Json template, together with a created CSV file. But the utility seems to fail each time. Looking at the console from foundry it appears to parse the first item in the CSV file correctly but fails on generating the item. I've attached a copy of my CSV, Template and the original item. Any help greatly appreciated.

[Swade weapon template.txt](https://github.com/SvenWerlen/fvtt-data-toolbox/files/5965396/Swade.weapon.template.txt)
[SWADEitem.txt](https://github.com/SvenWerlen/fvtt-data-toolbox/files/5965397/SWADEitem.txt)
[hold out pistols-csv.txt](https://github.com/SvenWerlen/fvtt-data-toolbox/files/5965400/hold.out.pistols-csv.txt)

**Comment by SvenWerlen** [comment]
Hi Raileth,

Preparing the JSON template could be quite tricky and is the main reason why the import would fail. I strongly recommend using a formatter like https://jsonformatter.org/ and to paste the data from the console to see why the generated JSON is invalid. In your case, I found a few issues:
* closing bracket `}` not matching any `{`
* `,` at the end of the "list"
* In the CSV, `FALSE` should be replaced by `false`

![image](https://user-images.githubusercontent.com/938060/107722709-8a474280-6cad-11eb-8f8c-f8b00739c76f.png)

Here are the files I used to successfully import your dataset.
[hold.out.pistols-csv.txt](https://github.com/SvenWerlen/fvtt-data-toolbox/files/5969351/hold.out.pistols-csv.txt)
[Swade.weapon.template.txt](https://github.com/SvenWerlen/fvtt-data-toolbox/files/5969352/Swade.weapon.template.txt)

Note : I found an issue in my code which makes the import fail when you have `"` in your data. Version `1.6.0` fixes it. Make sure to update your module.
